### DISCUSSION TITLE
Характеристики персонажа

### DISCUSSION THREAD

**Discussion by shar1zy** [octothorp-team/public-dobrograd#740] [Предложения]
Хотел бы предложить идею. она заключается в том что у персонажа могли бы быть характеристики, можно было бы создать в здании какой-нибудь спорт комплекс где можно было бы прокачивать силу, выносливость и тд. А когда прокачиваешь силу увеличивается урон с руки, а выносливость ты дольше бегаешь.

**Comment by pr0pk1** [comment]
некоторая система с хеликса/клокворка, где решает не твоя ролевая игра, а характеристики персонажа.
